### ISSUE TITLE
Disabling of Get XML and Post KVP 

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/ets-wcs10#20]
As part of our capability we are not supporting Get-XML and Post-KVP protocols. How can we configure team engine to disable their test cases execution?

**Comment by lgoltz** [comment]
As already described in #18 POST KVP is enabled if POST is activated. GET XML is not supported and not allowed by http.

**Comment by dstenger** [comment]
Decision: If POST KVP is optional, it should be possible to deactivate the protocol. Further investigation is required.

**Comment by dstenger** [comment]
The specification (OGC 05-076) states the following:
```
...
6.3.3 XML encoding

Clients may also encode requests in XML for transmission to the server using HTTP
GET or (more often) HTTP POST. The XML request must conform to the schema corre-
sponding to the chosen operation, and the client must send it to the URL listed for that
operation in the server’s Capabilities XML file, in accordance with HTTP POST [IETF
RFC 2616]).
...
```

Thus, if the service supports the POST DCPType, POST KVP must be a possible encoding.

--> No fix is required.

**Comment by bermud** [comment]
Proposed solution/decision

1- GetCapabilities advertises GET
2- check for GET KVP (if error or not responding, then)
3- check GET XML

1- GetCapabilities advertises POST
2- check for POST XML  (if error or not responding, then)
3- check POST  KVP

**Comment by lgoltz** [comment]
As far as I see the first block does not make sense. GET XML is not recommended as HTTP does not define the handling of a body for GET request. 

As service provider I would like to check all the features my services provides. Unfortunately it is not possible to get the information if XML and/or KVP is supported for POST requests from the capabilities. Sending a request and deciding dependent on the response (error or no response) is difficult to cause it is not clear if an error occurred cause of a unsupported requested binding or another failure. 

So I would prefer to add a checkbox to allow the user to select the bindings to test. Also the  subdivision into profiles (GET binding, POST/KVP binding, POST/XML binding) could be a solution.

**Comment by dstenger** [comment]
I agree and  propose to add a radio button making it possible to choose between ```POST/KVP binding``` and ```POST/XML binding```. Thus, always one of those options must be selected and is used if any POST encoding is advertised in capabilities. This solution has the drawback that mixed POST bindings are not supported by the test suite. But this is, in my opinion, a very rare case.
